ios/RCCDrawerController/RCCDrawerController.m

```diff
@@ -13,6 +13,9 @@ @implementation RCCDrawerController
 
 @synthesize drawerStyle = _drawerStyle;
 
+UIViewController *leftViewController = nil;
```

Comment:
Because leftViewController and rightViewController are defined locally in initWithProps as well as at a class level here, the class variables never get set. When you go to actually set the status of one of the drawers, the controller is therefore always nil (that caused me quite some confusion, as MMDrawerController doesn't handle that situation nicely and permanently disables interaction on the UI).